### PR TITLE
Replaced mmg from number and firetext from number with single from nu…

### PR DESCRIPTION
**Pull request by mannickutd**
…mber.

### CODE REVIEW

File: config_live.py [hunk 1]
```diff
@@ -14,14 +14,14 @@ class Live(Config):
...
     FIRETEXT_API_KEY = os.getenv("LIVE_FIRETEXT_API_KEY")
-    FIRETEXT_NUMBER = os.getenv("LIVE_FIRETEXT_NUMBER")
     TWILIO_AUTH_TOKEN = os.getenv('LIVE_TWILIO_AUTH_TOKEN')
     MMG_API_KEY = os.environ['LIVE_MMG_API_KEY']
     CSV_UPLOAD_BUCKET_NAME = 'live-notifications-csv-upload'
     STATSD_ENABLED = True
     STATSD_HOST = os.getenv('LIVE_STATSD_HOST')
     STATSD_PORT = os.getenv('LIVE_STATSD_PORT')
     STATSD_PREFIX = os.getenv('LIVE_STATSD_PREFIX')
+    FROM_NUMBER = os.getenv('FROM_NUMBER')
```
Comment (servingUpAces) [inline config_live.py]: This should os.getenv('LIVE_FROM_NUMBER')

File: config_staging.py [hunk 2]
```diff
@@ -14,10 +14,10 @@ class Staging(Config):
     VERIFY_CODE_FROM_EMAIL_ADDRESS = os.environ['STAGING_VERIFY_CODE_FROM_EMAIL_ADDRESS']
     NOTIFY_EMAIL_DOMAIN = os.environ['STAGING_NOTIFY_EMAIL_DOMAIN']
     FIRETEXT_API_KEY = os.getenv("STAGING_FIRETEXT_API_KEY")
-    FIRETEXT_NUMBER = os.getenv("STAGING_FIRETEXT_NUMBER")
     TWILIO_AUTH_TOKEN = os.getenv('STAGING_TWILIO_AUTH_TOKEN')
     MMG_API_KEY = os.environ['STAGING_MMG_API_KEY']
     CSV_UPLOAD_BUCKET_NAME = 'staging-notifications-csv-upload'
+    FROM_NUMBER = os.getenv('FROM_NUMBER')
```
Comment (servingUpAces) [inline config_staging.py]: This should os.getenv('STAGING_FROM_NUMBER')

**Comment by servingUpAces** [comment]
Environments have been updated - but the live and staging configs in the PR needs updating first

**Comment by mannickutd** [comment]
@servingUpAces Can you have a look to see this matches up to the credstash thanks.

**Comment by servingUpAces** [comment]
👍

**Pull request merged**